src/go.mod

```diff
@@ -3,6 +3,7 @@ module registry-stable
 go 1.21
 
 require (
+	github.com/google/go-cmp v0.5.9
```

Comment:
**(nitpick)** I feel that we could use either cmp, or testify, both feels overhead. WDYT?